### ISSUE TITLE
Version 3.4.3 fails in IE 11

### ISSUE THREAD

**Issue by iamthechad** [dbfannin/ngx-logger#126]
We're seeing this error in IE11 when trying to log with the latest version of this module:

(Amended log to remove project specifics)

```
TypeError: Unable to get property 'split' of undefined or null reference
 at NGXMapperService.getStackLine
 at NGXMapperService.prototype.getCallerDetails
 at NGXLogger.prototype._log
 at NGXLogger.prototype.warn
```

Based on [old MSDN information](https://web.archive.org/web/20140210004225/http://msdn.microsoft.com/en-us/library/windows/apps/hh699850.aspx), it looks like the error's stack doesn't get populated until it's thrown in IE11.

**Comment by dbfannin** [comment]
@iamthechad, I updated that code to fix the issue. I use a mac, so it's more difficult to test in IE. Could you verify the fix in IE for me?  ngx-logger@3.4.5

**Comment by iamthechad** [comment]
Looks like that did the trick! IE is back up and running. Thanks for super quick fix on this!
